packages/generators/utils/plugins.ts

```diff
@@ -1,14 +1,11 @@
-"use strict";
-
 /**
  *
  * Callable function with the initial plugins
  *
- * @param {Void} _ - void value
  * @returns {Function} An function that returns an array
  * that consists of the uglify plugin
  */
 
-module.exports = _ => {
+export default function(): string[] {
```

Comment:
`(_?:void)`

Reply:
It's not a promise, so `_` should be removed. Can we have a eslint rule?

Reply:
`_` on empty params please http://po-ru.com/diary/rubys-magic-underscore/

Reply:
That's Ruby, that treats them differently, what's your point?
Also, underscore is used as temporary placeholder in libraries such as Ramda, where it has a different meaning. My point is, what's the heff meaning of this underscore in this project? Where's documented? Why there's no eslint rule to tell me that the developer has to put in certain cases?

Reply:
Responded this internally. Needs an issue to be in the desc of the `update docs` issue